src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -3,85 +3,64 @@ title: Proof-of-stake (PoS)
 
-The committee has a time-frame in which to propose and validate a shard block. This is known as a "slot." Only one valid block is created per slot, and there are 32 slots in an "epoch." After each epoch, the committee is disbanded and reformed with different, random participants. This helps keep shards safe from committees of bad actors.
+## Fork choice {#fork-choice}
 
-#### Crosslinks {#rewards-and-penalties}
-
-Once a new shard block proposal has enough attestations, a "crosslink" is created which confirms the inclusion of the block and your transaction in the beacon chain.
-
-Once there's a crosslink, the validator who proposed the block gets their reward.
-
-#### Finality {#finality}
-
-In distributed networks, a transaction has "finality" when it's part of a block that can't change.
-
-To do this in proof-of-stake, Casper, a finality protocol, gets validators to agree on the state of a block at certain checkpoints. So long as 2/3 of the validators agree, the block is finalised. Validators will lose their entire stake if they try and revert this later on via a 51% attack.
-
-As Vlad Zamfir put it, this is like a miner participating in a 51% attack, causing their mining hardware to immediately burn down.
+When the network performs optimally and honestly, there is only ever one new block at the head of the chain, and all validators attest to it. However, it is possible for validators to have different views of the head of the chain due to network latency or because a block proposer has equivocated. Therefore, consensus clients require an algorithm to decide which one to favor. The algorithm used in proof-of-stake Ethereum is called LMD-GHOST, and it works by identifying the fork that has the greatest weight of attestations in its history.
```

Comment:
Should we link out to the Gasper page on some of these terms? ie. LMD-GHOST, or "inactivity leak" above?